src/test/resources/pages/instructorFeedbackResultsEditComment.html

```diff
@@ -876,9 +876,9 @@ <h4 class="modal-title">
                                 <div class="row" id="commentBar-1-0-1-1-1">
                                   <div class="col-xs-10">
                                     <span class="text-muted">
-                                      From: Teammates Test2 [Tue, 03 Apr 2012, 07:59 AM SGT] (last edited by Teammates Test at ${datetime.now} SGT)
+                                      From: Teammates Test2 [Tue, 03 Apr 2012, 07:59 AM SGT]
```

Comment:
Why the text get changed?

Reply:
Because I refactored tests. Tests for RGQ view were previously dependent upon Questions view in which we edited the comment.